app/views/styleguide/layout.html.erb

```diff
@@ -0,0 +1,113 @@
+<h1>Layout</h1>
+
+<h3 class="styleguide__subheading">Layout container</h3>
+<div class="l-constrained">
+  <div class="styleguide__example">
+    <h3>1 column, full width of container</h3>
+  </div>
+</div>
+
+<h3 class="styleguide__subheading">Code</h3>
+<xmp>
+  <div class="l-constrained">
+    <h3>1 column, full width of container</h3>
+  </div>
+</xmp>
+
+<h3 class="styleguide__subheading">1 Column Layout</h3>
+<div class="l-constrained">
+  <div class="l-1col">
+    <div class="styleguide__example">
+      <h3>1 column, full width of container</h3>
+    </div>
+  </div>
+</div>
+
+<h3 class="styleguide__subheading">Code</h3>
+<xmp>
+  <div class="l-constrained">
+    <div class="l-1col">
+      <h3>1 column, full width of container</h3>
+    </div>
+  </div>
+</xmp>
+
+<h3 class="styleguide__subheading">2 columns small on left</h3>
+<div class="l-constrained">
+  <div class="styleguide__example">
+    <div class="l-2col">
+      <div class="l-2col-side">
+        <h3>SIDE</h3>
+      </div>
+      <div class="l-2col-main">
+        <h3>MAIN</h3>
+      </div>
+    </div>
+  </div>
+</div>
+
```

Comment:
I am just wondering about naming on these "side" and "main" columns. Should we be reflecting the role of this content rather than its position in the layout, particularly in views other than desktop when this is not reflected in the layout (i.e. they are stacked rather than aligned).